### DISCUSSION TITLE
print user name instead of user <anonymous> into logs

### DISCUSSION THREAD

**Discussion by arturovivas** [authelia/authelia#4701] [Q&A] [answered]
Is there a way to configure authelia so that instead of giving the `<anonymous>` tag in the logs, it prints into the log the name of the user trying to access the resource? No matter if it is a user in the user DB or an unauthorised entity.

**Comment by james-d-elliott** [comment, marked as answer]
It only should print `<anonymous>` if they are unauthenticated. There is no way to determine the username of an unauthenticated user reliably.

**Reply by arturovivas** [reply]
Got it, thanks for the reply @james-d-elliott
